index.html

```diff
@@ -1,228 +1,188 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Consulting inc.</title>
     <link rel="stylesheet" href="style.css" />
 
-    <link href="//fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet" type="text/css" />
-
-
-</head>
+    <link
+      href="//fonts.googleapis.com/css?family=Open+Sans"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
 
-<body>
+  <body>
     <header>
-        <nav>
-            <img src="images/menu.svg" alt="" class="menu" />
-            <img src="images/logo1.jpg" alt="" class="desk-logo">
-            <div class="nav-links">
-                <ul>
-                    <li><a href="#">About</a></li>
-                    <li><a href="#">Program</a></li>
-                    <li><a href="#">Join</a></li>
-                    <li><a href="#">Sponsor</a></li>
-                    <li><a href="#">News</a></li>
-                    <li><a href="#" class="bold">BAKHTAR INC</a></li>
-                </ul>
-            </div>
-        </nav>
+      <nav>
+        <img src="images/menu.svg" alt="" class="menu" />
+        <img src="images/logo1.jpg" alt="" class="desk-logo" />
+        <div class="nav-links">
+          <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#" class="bold">BAKHTAR INC</a></li>
```

Comment:
please create a link that will take you to the about section on your desktop version
your design
![image](https://user-images.githubusercontent.com/60612329/191741650-cb4d31c9-e87c-48e3-94f3-182b3c7bc18c.png)